### PR TITLE
doc: update java studio docs

### PR DESCRIPTION
**Pull request by oldLady344**
- updated links in java and studio docs
- updated data sources for exchange
- updated nebula configuration parameters in ex-ug-import-sst.md
- updated ex-ug-import-steps.md

### CODE REVIEW

**Review by randomJoe211 — DISMISSED** [review]
LGTM

File: spark-connector/sc-ug-what-is-spark-connector.md [hunk 1]
```diff
@@ -1,15 +1,7 @@
...
-- Spark-connector Writer：目前正在开发中。
-
-## 贡献
-
-Spark-connector 是一个完全开源的项目，欢迎开源爱好者通过以下方式参与：
-
-- 前往 [Nebula Graph 论坛](https://discuss.nebula-graph.com.cn/ "点击前往“Nebula Graph 论坛") 上参与 Issue 讨论，如答疑、提供想法或者报告无法解决的问题
-- 撰写或改进文档
-- 提交优化代码
+- Writer：目前正在开发中。
```
Comment (Nicole00) [inline spark-connector/sc-ug-what-is-spark-connector.md:7]: writer可以用了呀

File: nebula-exchange/about-exchange/ex-ug-limitations.md [hunk 2]
```diff
@@ -35,6 +35,8 @@ Exchange 支持将以下格式或来源的数据转换为 Nebula Graph 能识别
   - Apache ORC
   - JSON
   - CSV
+- Elasticsearch
```
Comment (Nicole00) [inline nebula-exchange/about-exchange/ex-ug-limitations.md:38]: ElasticSearch先不加吧，等功能测过再加吧

File: spark-connector/sc-ug-what-is-spark-connector.md (hunk 1 above)
Comment (oldLady344) [reply spark-connector/sc-ug-what-is-spark-connector.md:7]: > writer可以用了呀

我会另开一个 pr 更新所有 Writer 相关的信息。

**Review by Nicole00 — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Pull request merged**